### PR TITLE
User management

### PR DESCRIPTION
**Pull request by kont-noor**
### Changes

* Add user management functionality
* Move itemable functionality to concern
* Itemable fixing (associations)

### CODE REVIEW

File: app/helpers/theblog/admin_helper.rb [hunk 1]
```diff
@@ -1,5 +1,34 @@
 module Theblog
   module AdminHelper
+    def formatted_value(item, attribute)
+      if attribute.is_a?(Hash)
+        association_name = attribute.keys.first
+        if model_associations.include?(association_name)
+          association = item.send(association_name)
+          if association.present?
+            if association.respond_to?(:any?)
```
Comment (skarm61) [inline app/helpers/theblog/admin_helper.rb]: for me - to many 'if'

Comment (skarm61) [inline app/helpers/theblog/admin_helper.rb]: I mean deep

**Comment by skarm61** [comment]
:+1:

File: app/helpers/theblog/admin_helper.rb (hunk 1 above)
Comment (skarm61) [inline app/helpers/theblog/admin_helper.rb]:
```
    def formatted_value(item, attribute)
      unless attribute.is_a?(Hash) 
        return raw(item.send(attribute))
      end

      association_name = attribute.keys.first
      association = item.try(association_name)
      return if association.nil?

      if association.respond_to?(:any?)
        raw(association.map { |el| print_association(el, attribute) }.join(', '))
      else
        print_association(association, attribute)
      end
    end
```

Comment (kont-noor) [inline app/helpers/theblog/admin_helper.rb]: updated

**Pull request merged**